### ISSUE TITLE
Copy button is not shown when markdown is indented

### ISSUE THREAD

**Issue by leovincentseles** [CaiJimmy/hugo-theme-stack#479]
## Describe the bug
![image](https://user-images.githubusercontent.com/36748589/151806661-7171f46d-392e-4356-9061-20e5bb92b596.png)
- In the markdown file, I indent the code section. After this modification, the copy button will disappear.

## Expected behavior
![image](https://user-images.githubusercontent.com/36748589/151807348-f154f590-82a1-413b-9707-4ec34a79741b.png)
![螢幕擷取畫面 2022-01-31 214934](https://user-images.githubusercontent.com/36748589/151806952-c3fa2901-abfa-4863-9f89-368c518a8d16.png)
- Without the markdown indentation, the copy button is shown correctly.
- I expect the copy button also appear when the markdown is indented.

## To Reproduce
Indent the code section in the markdown, the copy button is not shown.

## Screenshots
Add screenshots to help explain your problem.

## Environment
- **Hugo version** [e.g: 0.80.0]: 0.92.1
- **Hugo extended?**: Yes

If it's a UI issue, fill the following information:

 - OS: [e.g. iOS] Windows 11 (21H2)
 - Browser [e.g. chrome, safari]: chrome
 - Version [e.g. 22]: 97

## Content of config.yaml
```yaml
markup:
    goldmark:
        renderer:
            ## Set to true if you have HTML content inside Markdown
            unsafe: false
    tableOfContents:
        endLevel: 4
        ordered: true
        startLevel: 2

    ## See the documentation: https://gohugo.io/content-management/syntax-highlighting/
    highlight:
        codeFences: true
        guessSyntax: true
        hl_Lines: ""
        lineNoStart: 1
        lineNos: true   
        lineNumbersInTable: true
        noClasses: false
        tabWidth: 4
```

## Link to the demo site and/or source repository
- [My personal website](https://leovincentseles.github.io/posts/binary-search/)

**Comment by zhixuan2333** [comment]
I'm very sorry. Because this bug is from my pr.
#455

A short term solution is to set `.article-content > div.highlight` to `.article-content div.highlight`
https://github.com/CaiJimmy/hugo-theme-stack/blob/67d5156507acdf66cc132d3a1d59f2bb92486c8f/assets/ts/main.ts#L65

**Comment by leovincentseles** [comment]
@zhixuan2333 Thanks a lot for the provided solution!.

**Comment by CaiJimmy** [comment]
Should be fixed now with #487
